.circleci/config.yml

```diff
@@ -42,6 +45,7 @@ workflows:
             branches:
               ignore:
                 - gh-pages
+                - master
```

Comment:
I think this will break things, since we need the build to run on the master branch. It might make sense to make a new job called `deploy-preview` and then do the branch filter on that job instead of on the `build-website` job

Reply:
@joeldenning take a look at the latest diff – I put a condition inside the existing `build-website` job.